frontend/src/views/website/runtime/php/extensions/operate/index.vue

```diff
@@ -51,6 +47,10 @@ import { MsgSuccess } from '@/utils/message';
 import { CreatePHPExtensions, UpdatePHPExtensions } from '@/api/modules/runtime';
 import i18n from '@/lang';
 import { Runtime } from '@/api/interface/runtime';
+import { GlobalStore } from '@/store';
+
+const globalStore = GlobalStore();
+const phpDocURL = globalStore.isIntl ? `/user_manual/websites/runtime_php/` : '/user_manual/websites/php/#php_1';
 
 const open = ref(false);
 const operate = ref('create');
```

Comment:
The changes look mostly acceptable. Here's a quick review:

1. **Inline link removal**: You've removed the inline link and replaced it with `el-link`. This should be fine if you're using Vue 3 and have properly set up the `setup()` function.

2. **Global store usage**:
   - The variable `globalStore` is declared within a computed property which isn't necessary.
   - The URL computation can be simplified to just concatenate constants if they don't change during runtime.
3. **Dynamic text translation**: It seems there might be an issue with translating `$t('php.toExtensionsList')`. Ensure that `i18n.js` has the appropriate translations for this key.
4. **Variable naming**: While `glovalStore` is technically correct, it could be more descriptive like `runtimeStore`.

Overall, these adjustments are small and shouldn't affect functionality significantly unless other parts of your application depend on them correctly.